lib/magma/project.rb

```diff
@@ -47,6 +47,20 @@ def load_project
         require_files('loaders')
         require_files('metrics')
       end
+
+      load_model_attributes
+    end
+
+    def load_model_attributes
+      model_attributes = Magma.instance.db[:attributes].
+        where(project_name: @project_name.to_s).
+        to_a.
+        group_by { |attribute| attribute[:model_name].to_sym }
+
+      model_attributes.each do |model_name, attributes|
+        model = models[model_name]
+        model.load_attributes(attributes)
```

Comment:
might this be null?

Reply:
Are you saying the model might be null? It's possible, but seems unlikely. I can add a check though to make sure this doesn't blow up if there isn't a model for the attribute.

Reply:
@graft can you clarify what you mean here?

Reply:
If the attributes table (+ a models table if we add one) ever diverge from the database or the class definition we would have a conflict here (if, e.g., we remove one of the models from the magma-ipi repo, but there is still a definition for it in the attributes table). When we boot and try to attach those attributes to the non-existent model, this will fail; we should guard against this possibility of divergence by raising the appropriate error here to warn about the mismatch.

Reply:
Gotcha, thanks for explaining 😄. If this happens, we'll raise a `Magma::Project::LoadError`. See 94eed4c2e6939bbffa71b251ee1687eda32002cd.